src/main/java/org/terasology/blockNetwork/traversal/BreadthFirstTraversal.java

```diff
@@ -24,5 +24,13 @@
  * @param <V> Value passed from node visited into nodes that are outgoing from that node.
  */
 public interface BreadthFirstTraversal<T, U, V> {
+    /**
+     * Traverses the given node in the graph.
+     * This method is invoked for each node in a traversal as directed by the traversal definition and the traversal result.
+     * 
+     * @param node The type of nodes in the graph.
```

Comment:
I'm not sure what you are doing here.

`node` is *not* the type of nodes in the graph, it is the node that is currently being traversed.

I get the feeling that you've essentially blindly copied the class level javadocs on this class, without even reading them once. I seriously did not expect that from you.

Reply:
In response to an earlier, unedited version of this comment: I'm sorry if it seemed like I was being rude, I definitely did not want it to come out that way.

Reply:
You don't have to be. You were completely right, and this kind of sloppiness should not be expected. I apologize; once I have access to my work environment in a few hours, I can guarantee you that my work will return to the level it previously was. I shall take a good look at everything again.